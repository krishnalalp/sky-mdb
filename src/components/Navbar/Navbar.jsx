import React from "react";
import { useHistory } from "react-router-dom";
import { Search } from "../Searchbar/Searchbar";
import logo from "../../assets/img/sky_logo.png";
import "./Navbar.scss";

export function Navbar() {
  const history = useHistory();

  const handleClick = () => {
    history.push("/");
  };

  return (
    <div className="navbar">
      <div
        className="logo"
        role="button"
        tabIndex={0}
        onClick={() => handleClick()}
        onKeyDown={() => handleClick()}
      >
        <img alt="sky_logo" src={logo} />
      </div>
      <Search />
    </div>
  );
}
