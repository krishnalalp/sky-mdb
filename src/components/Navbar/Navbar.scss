.navbar {
  display: flex;
  align-items: center;
  flex: 1;
}

.logo img {
  height: 80px;
  cursor: pointer;
}
