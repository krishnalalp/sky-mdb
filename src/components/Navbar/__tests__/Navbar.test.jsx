import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { Navbar } from "../Navbar";

test("Navbar component rendered", () => {
  const renderer = new ShallowRenderer();

  renderer.render(<Navbar />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
