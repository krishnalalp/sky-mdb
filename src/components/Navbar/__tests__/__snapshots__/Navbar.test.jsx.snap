// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component rendered 1`] = `
<div
  className="navbar"
>
  <div
    className="logo"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="button"
    tabIndex={0}
  >
    <img
      alt="sky_logo"
      src="sky_logo.png"
    />
  </div>
  <Search />
</div>
`;
